win32ss/user/ntuser/menu.c

```diff
@@ -4617,7 +4617,7 @@ BOOL WINAPI IntTrackPopupMenuEx( PMENU menu, UINT wFlags, int x, int y,
           if (!(wFlags & TPM_NONOTIFY))
           {
              co_IntSendMessage( UserHMGetHandle(pWnd), WM_UNINITMENUPOPUP, (WPARAM)UserHMGetHandle(menu),
-                                            MAKELPARAM(0, IS_SYSTEM_MENU(menu)) );
+                                            MAKELPARAM(0, IS_SYSTEM_MENU(menu) ? MF_SYSMENU : 0) );
```

Comment:
Question: Are those modifications for `WM_UNINITMENUPOPUP` are for implementing the comment made in
https://learn.microsoft.com/en-us/windows/win32/menurc/wm-uninitmenupopup ? Namely:
> lParam
>
> The high-order word identifies the menu that was destroyed. Currently, this parameter can only be MF_SYSMENU (the window menu).

Reply:
Yes, they are implemeting this comment